src/main/java/com/microsoft/sqlserver/jdbc/dataclassification/SensitivityClassification.java

```diff
@@ -8,20 +8,54 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.microsoft.sqlserver.jdbc.SQLServerException;
+
 
 /**
- * Provides the functionlity to retrieve Sensitivity Classification data as received from SQL Server for the active
+ * Provides the functionality to retrieve Sensitivity Classification data as received from SQL Server for the active
  * resultSet
  */
 public class SensitivityClassification {
+
+    public enum SensitivityRank {
```

Comment:
SensitivityRank used to not be visible to public, but now is it. Is this intended?

Reply:
yes this is so users can check the values